backend/alembic/versions/0ffdc7e11ff6_create_table.py

```
@@ -12,19 +12,21 @@
-revision: str = 'cb2788081aa3'
-down_revision: Union[str, None] = '6d723cc4e39e'
+revision: str = '0ffdc7e11ff6'
+down_revision: Union[str, None] = '6bdd563d49e1'
 branch_labels: Union[str, Sequence[str], None] = None
 depends_on: Union[str, Sequence[str], None] = None
 
 
 def upgrade() -> None:
+    """Upgrade schema."""
     # ### commands auto generated by Alembic - please adjust! ###
     pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)